users/postgres/users.go

```diff
@@ -141,26 +249,58 @@ func (m dbMetadata) Value() (driver.Value, error) {
 }
 
 type dbUser struct {
-	ID       string     `db:"id"`
-	Email    string     `db:"email"`
-	Password string     `db:"password"`
-	Metadata dbMetadata `db:"metadata"`
+	ID       string `db:"id"`
+	Owner    string `db:"owner"`
+	Email    string `db:"email"`
+	Password string `db:"password"`
+	Metadata []byte `db:"metadata"`
 }
 
-func toDBUser(u users.User) dbUser {
+func toDBUser(u users.User) (dbUser, error) {
+	data := []byte("{}")
+	if len(u.Metadata) > 0 {
+		b, err := json.Marshal(u.Metadata)
+		if err != nil {
+			return dbUser{}, errors.Wrap(errMarshal, err)
+		}
+		data = b
+	}
+
 	return dbUser{
 		ID:       u.ID,
 		Email:    u.Email,
 		Password: u.Password,
-		Metadata: u.Metadata,
-	}
+		Metadata: data,
+	}, nil
 }
 
-func toUser(dbu dbUser) users.User {
+func toUser(dbu dbUser) (users.User, error) {
```

Comment:
Why is this function called `toUser()`? What does this `to` prefix mean?

Similarly - why is `toDBUser()` called like this?

Maybe there is a reason, but it is cryptic - if it is valid please put a comment above function names.

Reply:
OK, this: https://github.com/mainflux/mainflux/blob/master/things/postgres/things.go#L344 shows consistency, so I'll approve.